src/layers/core/path-layer/path-layer.js

```diff
@@ -41,15 +41,41 @@ export default class PathLayer extends Layer {
     });
 
     const {attributeManager} = this.state;
+    const noAlloc = true;
     attributeManager.addDynamic({
-      indices: {size: 1, update: this.calculateIndices, isIndexed: true},
-      positions: {size: 3, update: this.calculatePositions},
-      leftDeltas: {size: 3, update: this.calculateLeftDeltas},
-      rightDeltas: {size: 3, update: this.calculateRightDeltas},
-      directions: {size: 1, update: this.calculateDirections},
-      colors: {size: 4, type: GL.UNSIGNED_BYTE, update: this.calculateColors},
-      pickingColors: {size: 3, type: GL.UNSIGNED_BYTE, update: this.calculatePickingColors,
-        noAlloc: true}
+      indices: {
+        size: 1,
+        isIndexed: true,
+        update: this.calculateIndices, noAlloc
```

Comment:
1) lots of missing accessors. 
2) I really think we should try to keep the attribute definitions on single lines. We have increased the max length in the linter to 100, so no need to break on 80 characters.